routemaster/models/subscriber.rb

```diff
@@ -79,6 +79,28 @@ def uuid
       _attributes['uuid']
     end
 
+    def health_points
+      _attributes.fetch('health_points', '100').to_i
+    end
+
+    def change_health_by(offset)
+      new_value = _redis_lua_run(
+        'subscriber_change_health_by',
```

Comment:
Should we load the script and execute by sha1 rather than sending the script each time?